Installation/test/Installation/CMakeLists.txt

```diff
@@ -83,15 +83,6 @@ if(WITH_CGAL_Qt5)
   endif()
 endif()
 
-if(WIN32 OR CMAKE_SYSTEM_NAME STREQUAL Windows)
-  add_executable(display_dll_version_info display_dll_version_info.cpp)
-  target_link_libraries(display_dll_version_info version)
-  add_executable(test_gmp_mpfr_dll test_gmp_mpfr_dll.cpp)
-  target_link_libraries(test_gmp_mpfr_dll version)
-  cgal_add_test(test_gmp_mpfr_dll)
-  add_to_cached_list(CGAL_EXECUTABLE_TARGETS test_gmp_mpfr_dll)
-endif()
-
```

Comment:
Please do not remove the test `test_gmp_mpfr_dll`.

Reply:
It is not removed, this block is already present a few lines before, it has probably been duplicated during a merge.